### PR TITLE
Update testnet-kpis.md

### PR DESCRIPTION
**Pull request by GavT**
### Why this change is needed

Please provide a description and a link to the underlying ticket

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: planning/testnet-kpis.md [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
-|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks||
-| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks||
-| Wallet extension downloads| Good proxy for the number of active users. Straightforward to capture.| GitHub|Number of wallet extension downloads in the last 4 weeks||
+| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks|99%|
+| Number of wallets connected to Obscuro Gateway| Good proxy for the number of active users. Straightforward to capture.| Datadog? |Number of daily connections|500|
+| Number of transactions| Typical guage of the amount of activity on testnet (even though it can be gamed).| Datadog? |Number of daily transactions|2000|
```
Comment (CaisManai) [inline planning/testnet-kpis.md:16]: We should add a line to capture the number of RPC requests and see how many we can handle without falling over

File: planning/testnet-kpis.md [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
+| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks|20|
 
 ## Discounted KPIs
-These KPIs were considered and discounted.
+These KPIs were considered and discounted for now.
 
+- Wallet extension downloads
+    - Good proxy for the number of active users.
+    - Straightforward to capture.
+    - Discounted because Obscuro Gateway is the recommended wallet connection method.
```
Comment (CaisManai) [inline planning/testnet-kpis.md:31]: Agreed. It almost becomes a negative KPI. Hopefully the wallet extension gets downloaded 0 times which will show people trust our gateway.

File: planning/testnet-kpis.md [hunk 3]
```diff
@@ -1,29 +1,40 @@
...
 - Ported Solidity dApps– original vs fork
     - One of Obscuro’s key promises is that existing Solidity dApps will just work on Obscuro. This KPI helps track whether Obscuro is fulfilling this promise. The reason for differentiating between original fork e.g. Uniswap vs PancakeSwap is it further provides insight into whether original teams believe in Obscuro.
     - Discounted because very difficult to determine of the dApps on Testnet which have been ported.
-- Number of transactions
-    - Gauge the amount of activity on Testnet.
-    - Discounted because it can be gamed too easily leading to misleading results.
 - Number of nodes
     - Nodes secure the network and track general interest in Obscuro. This KPI also provides an input into the staking function and has an impact on tokenomics.
-    - Discounted for now because too early in the lifecycle of Testnet to bring nodes online.
+    - Discounted for now because Obscuro Labs will run the nodes. When validator nodes can be run by others, this KPI will be introduced.
```
Comment (CaisManai) [inline planning/testnet-kpis.md:37]: I wonder if we should track the number of applications to become a node operator at this stage through another typeform form

**Review by CaisManai — CHANGES_REQUESTED** [review]
Feedback left. Let me know what you think.

File: planning/testnet-kpis.md [hunk 4]
```diff
@@ -1,29 +1,40 @@
...
-| Number of end user accounts| Captures users using testnet in general and testing new dApps and shows the pipeline from wallet extension download to actively using testnet | TBD | Number of new end user account addresses in the last 4 weeks ||
-|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks||
-| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks||
-| Wallet extension downloads| Good proxy for the number of active users. Straightforward to capture.| GitHub|Number of wallet extension downloads in the last 4 weeks||
+| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks|99%|
```
[the remark this answers is not in the document]
Comment (GavT) [reply planning/testnet-kpis.md:14]: There is a price / UX tradeoff.
99.999% allows 5 minutes and 15 seconds of downtime a year
99.99% could be unavailable 52 minutes and 36 seconds per year.
99.9% allows 8 hours and 46 minutes of downtime per year.
For testnet, 99.9% is probably achievable without too much outlay (load balancers, multiple servers, follow-the-sun ops)
Suggest we shoot for 99.9% on Testnet, but 99.999% on Mainnet

File: planning/testnet-kpis.md (hunk 1 above)
Comment (GavT) [reply planning/testnet-kpis.md:16]: Agreed. Presumably this is also an indicator of the amount of interaction.

File: planning/testnet-kpis.md (hunk 3 above)
Comment (GavT) [reply planning/testnet-kpis.md:37]: We already have a list of people who want to operate a node - are you saying it could be stale?

Comment (CaisManai) [reply planning/testnet-kpis.md:38]: Yes, it's coming up to 1.4 years now since we captured

**Review by CaisManai — APPROVED** [review]

**Pull request merged**